src/Task/StaticCacheFullBuildTask.php

```diff
@@ -2,22 +2,96 @@
 
 namespace SilverStripe\StaticPublishQueue\Task;
 
+use SilverStripe\Control\Director;
 use SilverStripe\Control\HTTPRequest;
 use SilverStripe\Dev\BuildTask;
+use SilverStripe\ORM\DataList;
+use SilverStripe\ORM\FieldType\DBDatetime;
 use SilverStripe\StaticPublishQueue\Job\StaticCacheFullBuildJob;
+use Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor;
+use Symbiote\QueuedJobs\Services\QueuedJob;
+use Symbiote\QueuedJobs\Services\QueuedJobService;
 
 class StaticCacheFullBuildTask extends BuildTask
 {
 
     /**
-     * Implement this method in the task subclass to
-     * execute via the TaskRunner
+     * Queue up a StaticCacheFullBuildJob
+     * Check for startAfter param and do some sanity checking
      *
      * @param HTTPRequest $request
      * @return
      */
     public function run($request)
     {
         $job = new StaticCacheFullBuildJob();
+        $signature = $job->getSignature();
```

Comment:
The signature can be effected by the job's `jobData`, or not for the full build job?

Reply:
I'd consider doing something like:

```php
$ids = JobDescriptor::get()->getIDList();
$id = QueuedJobService::singleton()->queueJob($job);
if (in_array($id, $ids)) {
    return "already in queueu";
}
```

Rather than duplicating some of the `QueuedJobService` duplicate detection logic